src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -2471,6 +2471,12 @@ well as menu structures (for main menu and popup menus).
         windowMode="main"
         desc="Clear console"/>
 
+   <cmd id="clearBuild"
+        buttonLabel=""
+        label="Clear Build"
```

Comment:
This label appears in the Command Palette; probably should be a bit more descriptive? Maybe: "Clear Build Pane Output"?